rec_center_model/src/model/entities/ProfessorStudent.java

```diff
@@ -9,19 +9,55 @@
 @Entity
 @DiscriminatorValue(value = CustomerType.Values.ProfessorStudent)
 public class ProfessorStudent extends Customer {
-	@Column(name = "enter_date")
+	@Column(name = "enter_date", nullable = false)
 	protected Date enterDate;
 
-	@Column(name = "grad_date")
+	@Column(name = "grad_date", nullable = false)
 	protected Date gradDate;
 
+	@Column(nullable = false)
 	private String major;
 
+	@Column(nullable = false)
```

Comment:
Shouldnt this be nullable? Not everybody has a minor.

Reply:
What is the difference between the empty string and null here?
We could just as well use `""` to mean "no minor".

Reply:
Cool I’m convinced! Approved